src/internal/minikubetestenv/deploy.go

```diff
@@ -322,6 +364,9 @@ func putRelease(t testing.TB, ctx context.Context, namespace string, kubeClient
 	if opts.Loki {
 		helmOpts = union(helmOpts, withLokiOptions(namespace, int(pachAddress.Port)))
 	}
+	if !(opts.Version == "" || strings.HasPrefix(opts.Version, "2.3")) {
```

Comment:
should we add a TODO to remove this once 2.4 hits?

Reply:
For the time being I opened CORE-654 to remind us.